src/features/Search/useQueryParameter.js

```diff
@@ -0,0 +1,11 @@
+import { useLocation } from "react-router-dom";
```

Comment:
Wydaje mi się, że pliki queryParameter  były plikami wyniesionymi w src,  a nie głębiej.

Reply:
masz racje, powinnam wyniesc wyzej, dlatego ze korzystam z niego zarówno w Search jak i w MovieList, a dokladnie to samo bedzie w PeoplePage, takze powinnam wyniesz do features, bo tylko pliki w features korzystaja z tych hookow